proto-build/src/main.rs

```diff
@@ -204,10 +206,18 @@ fn copy_generated_files(from_dir: &Path, to_dir: &Path) {
 fn copy_and_patch(src: impl AsRef<Path>, dest: impl AsRef<Path>) -> io::Result<()> {
     let contents = fs::read_to_string(src)?;
 
+    // For undetermined reasons `prost-build` is outputting this path for types
+    // defined in the `tendermint-proto` crate. Patch them to use the real crate.
+    // See: https://github.com/cosmos/cosmos-rust/issues/15
+    // TODO(tarcieri): mega hax, find a real solution
+    let contents = contents
+        .replace(MALFORMED_TENDERMINT_PROTO_CRATE_PATH, "tendermint_proto")
+        .replace("super::tendermint_proto", "tendermint_proto");
```

Comment:
For whatever reason when I run proto-build locally the output is attempting to use relative paths to access types defined in the tendermint-proto crate, as I noted in #15.

I would definitely like to find a better solution than this before merging because as the comment notes it's mega hax. However, this was enough to at least make the output reproducible on my end.

Reply:
I'm not happy with it either, but it seems to be the best solution around right now. Lets go for it.